src/app/shared/container/container.component.html

```diff
@@ -0,0 +1,3 @@
+<p class="container">
+  <ng-content></ng-content>
+</p>
```

Comment:
in this case separate component is unnecessary, you can use this <p> in the template